src/components/Sidebar/drawer-list.tsx

```
@@ -0,0 +1,207 @@
+                                                </ListItemIcon>
+                                                <StyledListItemText
+                                                    primary={item.name}
+                                                />
+                                                {item.count ? (
+                                                    <Tooltip
+                                                        title={
+                                                            'You have ' +
+                                                            item.count +
+                                                            (item.count == 1
+                                                                ? ' message.'
+                                                                : ' messages.')
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)